optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/employee/EmployeeRestService.java

```diff
@@ -55,27 +53,27 @@ Employee getEmployee(@ApiParam(required = true) @PathParam("tenantId") Integer t
     @POST
     @Path("/add")
```

Comment:
This wasn't changed in this PR, but I'm wondering isn't it more idiomatic to make this a POST against "/tenant/{tenantId}/employee/"?